### ISSUE TITLE
Add support for multiple group by time intervals

### ISSUE THREAD

**Issue by lexh** [grafana/grafana#2647]
This may only apply to InfluxDB, I haven't used Grafana with anything else recently. 

It would be nice to be able to override the time interval group by clause on a per query basis.

Specific use case: I want one metric graphed with high granularity (10s intervals) alongside a different metric graphed in broader strokes (1m intervals). The second metric is noisier and this would make it easier for me to parse the graph.

I think it would be nice to retain the current feature--manually setting something very similar to a template variable, which applies to all queries by default--but to also allow the group by time interval to be edited by hand for each specific query if desired. As Grafana makes API calls for each query in a graph, it seems like this should be within reach.

TLDR: I want the ability to edit `$interval` in the following screenshot:

![image](https://cloud.githubusercontent.com/assets/4326548/9626315/815d751c-5111-11e5-9e36-e5e469347e23.png)

**Comment by torkelo** [comment]
yes, this should be implemented!

**Comment by lexh** [comment]
Perhaps it provides a drop down with the common group by intervals available + a "custom" option.

**Comment by torkelo** [comment]
new query editor lets you specify group by time interval for each query.

![image](https://cloud.githubusercontent.com/assets/10999/9761623/3f6b4812-56fe-11e5-9ce7-c61deb2f25ae.png)
